exercises/es/test_general.py

```diff
@@ -50,4 +51,4 @@ def test_03_16_02_predictions(nlp):
         "especializada en sándwiches de pollo."
     )
     doc = nlp(text)
-    assert [ent.text for ent in doc.ents] == ["Chick", "College Park", "Georgia"]
\ No newline at end of file
+    assert [ent.text for ent in doc.ents] == ["Chick-fil-A", "College Park", "Georgia"]
```

Comment:
This is nitpicky, but just adding a newline at the end of the file, for consistency. (That red icon there indicates that there is none.) It's kind of a convention and if you're using Visual Studio Code, you can configure it to automatically add newlines at the end of files.

```suggestion
    assert [ent.text for ent in doc.ents] == ["Chick-fil-A", "College Park", "Georgia"]
    
```